### PR TITLE
added OCI

### PR DESCRIPTION
**Pull request by adamsj-oci**

### CODE REVIEW

File: data/america.yaml [hunk 1]
```diff
@@ -557,3 +557,9 @@
   long: -71.111465
   description: Master's Student in Northeastern University passionately working on Autonomous Mobile Robotics
   link: https://github.com/veeraragav
+- name: Object Computing, Inc
+  type: company
+  lat: 38.673867
+  long: -90.453198
+  description: Object Computing, Inc. offers ROS2, DDS, Cloud, AI/ML, blockchain, and other areas.
+
```
Comment (DLu) [inline data/america.yaml]:
```suggestion
```

**Comment by adamsj-oci** [comment]
Let me know if there are any other issues.

**Comment by DLu** [comment]
Somehow you ended up duplicating a couple of the existing entries when you merged. Can you open a clean PR?

**Comment by adamsj-oci** [comment]
Ok. I'll clean it up. Let's just close this PR.

**Pull request closed without merging**